tex/main.tex

```diff
@@ -56,14 +72,85 @@ \section{Model}
+
+This model allows for heterogeneity as we have a fitness function dependent on the individual, not dependent on the action taken. Therefore, we can give different individuals of the same action type a different fitness by passing different attributes to the fitness function itself. We will see examples of this in future sections.\\
+
+As this forms a Markov chain, we can therefore define a transition matrix \(T\) for any given state space. This can be used to show the transition probabilities between any two given states, and therefore can be used to find the fixation probabilities for any given starting state by the following method:\\
+
+Given a transition matrix \(T\), we can calculate the state distribution after \(k\) iterations of the Moran process using the formula \(T^k\)  \cite{Markov_Chain}. Therefore, as \(k \to \infty\), we will clearly acquire a distribution that tends towards the absorption probabilities for a given starting state, as the process will stabilise only if we enter an absorbing state (while subsets of \(S\) of states may have extremely high probabilities of transitioning to each other, the only way to have certainty of not transitioning away from a state or pair of states is to be in one of the previously defined absorbing states.)\\
+Thus, our absorption probabilities starting at are given by the non-zero entries in the matrix \(\lim_{k \to \infty} T^k\)
+
+\section{A Classic Public Goods Game}
+\subsection{The Standard Model}
+
+In this section, we shall see how this model can be applied to the classical public goods game. In this game, individuals choose whether or not to contribute to a public resource pool. In the end, the total resource is multiplied by some factor \(r > 1\) and distributed equally between each player. This model often encourages a behaviour known as ``free-riding" \cite{Climate Clubs}, where players refuse to contribute to the pool and simply take the benefit provided by other individuals' contribution. The classic problem of a public goods game is to provide an environment where such free-riding is not a profitable strategy - or rather, is less profitable than contributing to the public good.\\
+
+In our model, we can simulate a public goods game by providing the following payoff function:\\
+\begin{large}
+\begin{equation}
+    B(v_i) = \frac{r\sum_{j=1}^N{C_{v_j}}}{N} - C_{v_i}
+\end{equation}
+\end{large}\\
```

Comment:
Remove all `large` calls (don't mess around with fonts, when it's prepared for publications that's controlled elsewhere).